tests/playwright-test/golden.spec.ts

```diff
@@ -65,6 +65,70 @@ Line7`,
   expect(result.output).toContain('Line7');
 });
 
+test('should write detailed failure result to an output folder', async ({runInlineTest}, testInfo) => {
+  const result = await runInlineTest({
+    'a.spec.js-snapshots/snapshot.txt': `Hello world`,
+    'a.spec.js': `
+      const { test } = pwt;
+      test('is a test', ({}) => {
+        expect('Hello world updated').toMatchSnapshot('snapshot.txt');
+      });
+    `
+  });
+  expect(result.exitCode).toBe(1);
+  expect(result.output).toContain('Snapshot comparison failed:');
+  expect(result.output).toMatch(/ {4}Expected: .+?snapshot-expected\.txt/);
+  expect(result.output).toMatch(/ {4}Received: .+?snapshot-actual\.txt/);
+  expect(result.output).not.toMatch(/ {8}Diff: .+?snapshot-diff\.txt/);
```

Comment:
Why no diff in this case?

Reply:
There's just none. I've checked it out and it's because only `compareImages` returns the `diff` field, `compareText` and `compareBuffers` don't  have one.  
Should I create a separate test with the `test.fixme` annotation?

Reply:
Oh, I see! I think that diff for text snapshot might be useful, so file an issue if you feel like it. No need for a fixme test just yet.